src/js/types/database_info.ts

```diff
@@ -0,0 +1,10 @@
+export interface DatabaseInfo {
+  /// The AIRAC cycle that this database is.
+  ///
+  /// e.g. `2313` or `2107`

```

Comment:
It'd be great if we could stick to JSDoc on the TS side - that way the comments propagate in the editor/IDE 👌

Reply:
Aha yeah I copied it from my rust writing and forgot to update, I did for some other structs, just not this one apparently